### PR TITLE
Add new permission for Android sensors

### LINKED ISSUE #1904: Android 12 and blank white screen for taking photos
When I try to take a photo the camera appears with a blank white screen.

### PR DESCRIPTION
**Pull request by tomasMizera**
Android introduced new permission for sensors that we use during camera capture. Since we did not declare them, camera activity failed with white screen.

Should fix #1904 - @saberraz can you please confirm that?

### CODE REVIEW

File: app/android/AndroidManifest.xml [hunk 1]
```diff
@@ -110,6 +110,9 @@
     <!-- Camera permissions -->
     <uses-permission android:name="android.permission.CAMERA"/>
 
+    <!-- We are reading sensors that from API 31 needs additional permission -->
+    <uses-permission android:name="android.permission.HIGH_SAMPLING_RATE_SENSORS" />
```
Comment (PeterPetrik) [inline app/android/AndroidManifest.xml:114]: https://developer.android.com/reference/android/Manifest.permission#HIGH_SAMPLING_RATE_SENSORS

how this connects to camera at all?

Comment (tomasMizera) [reply app/android/AndroidManifest.xml:114]: We are reading OrientationSensor before taking a picture (for EXIF), here: https://github.com/lutraconsulting/input/blob/master/app/android/src/uk/co/lutraconsulting/CameraActivity.java#L49

**Comment by tomasMizera** [comment]
Closing, this one is more nasty. See comment in the ticket

**Comment by saberraz** [comment]
I cannot see any comments @tomasMizera in the ticket?

**Pull request closed without merging**